README.md

```diff
@@ -916,6 +917,22 @@ To exclude a property, write:
 
 **The property is excluded for all doc types.** Use a name that is unambiguous for such properties.
 
+### Managing the pages tree without workflow
+
+With the normal `apostrophe-workflow` behavior, when you move a page in the page tree modal, it only moves the draft one. Later, when committing, it will move the live one too. The draft document is moved relatively to a target (it can be `before`, `after` or `inside`), the live one is moved relatively to the live version of this target.
+
+In some cases, when moving too much nested pages without committing each one before, it can generate mismatches between draft and live pages trees and lead to confusion.
+
+To avoid this behavior and move draft and live pages at the same time, you can pass this flag to you workflow config:
+
+```javascript
+'apostrophe-workflow': {
+  syncPagesMoves: true,
+}
+```
+
+:warning: It won't commit it, it will just update properties linked to the pages tree like `path`, `rank`, `level` and `workflowMoved`. So the slug, for example, will be updated only when committing.
```

Comment:
This isn't great, but I don't see an alternative.

Reply:
How about just: "this will only commit the new position in the tree of a moved page. It will not commit any other properties of the page, such as the slug or title."

I do want to use the word "commit" here to go with the new name for the option.

Reply:
Isn't it what you did for A3 ? Repeating only moves automatically ?
We can use commit but I really find it confusing since it's not a commit as a user could expect it.

Reply:
Modified